Octokit/Models/Response/EmailAddress.cs

```diff
@@ -25,24 +27,54 @@ public EmailAddress(string email, bool verified, bool primary)
         public string Email { get; protected set; }
 
         /// <summary>
-        /// true if the email is verified; otherwise false
+        /// True if the email is verified; otherwise false
         /// </summary>
         public bool Verified { get; protected set; }
 
         /// <summary>
-        /// true if this is the users primary email; otherwise false
+        /// True if this is the users primary email; otherwise false
         /// </summary>
         public bool Primary { get; protected set; }
 
+        /// <summary>
+        /// "private" or "public" if the email address is the primary;
+        /// otherwise null
+        /// <summary>
+        public StringEnum<EmailVisibility> Visibility { get; protected set; }
+
         [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode",
             Justification = "Used by DebuggerDisplayAttribute")]
         internal string DebuggerDisplay
         {
             get
             {
                 return string.Format(CultureInfo.InvariantCulture,
-                    "EmailAddress: Email: {0}; Primary: {1}, Verified: {2}", Email, Primary, Verified);
+                    "EmailAddress: Email: {0}; Primary: {1}, Verified: {2}, Visibility: {3}", Email, Primary, Verified, Visibility);
             }
         }
     }
+
+    /// <summary>
+    /// Represents the visibility of an email address.
+    /// </summary>
+    public enum EmailVisibility
+    {
+        /// <summary>
+        /// Primary email address and is public
+        /// </summary>     
+        [Parameter(Value = "public")]
+        Public,
+
+        /// <summary>
+        /// Primary email address and is private
+        /// </summary> 
+        [Parameter(Value = "private")]
+        Private,
+
+        /// <summary>
+        /// Email is not the primary address
+        /// </summary>
+        [Parameter(Value = "null")]
```

Comment:
Need to remove this `All` option from the enum as it wont deserialize correctly anyway, plus we can make the property above nullable, to handle this case

Reply:
Done, let me know how these changes look. Thanks for the continued help on this!